utils/doclint/check_public_api/index.js

```diff
@@ -205,7 +205,8 @@ function compareDocumentations(actual, expected) {
       [/Handle\<R\>/g, 'JSHandle'],
       [/JSHandle\<Object\>/g, 'JSHandle'],
       [/object/g, 'Object'],
-      [/Promise\<T\>/, 'Promise<Object>']
+      [/Promise\<T\>/, 'Promise<Object>'],
+      [/Readable/, 'stream.Readable']
```

Comment:
If we leave this as `Readable`, it shouldn't need any special casing or overrides.